resources/resources/software/aws/toolkits/resources/MessagesBundle.properties

```diff
@@ -698,14 +698,14 @@ ecs.execute_command_disable_failed=Execute Command could not be disabled for {0}
 ecs.execute_command_disable_in_progress=Cannot execute action since Execute Command is being disabled for {0}
 ecs.execute_command_disable_progress_indicator_message=Disabling Command Execution for {0}
 ecs.execute_command_disable_success=Execute Command for {0} has been disabled.
-ecs.execute_command_disable_warning=Disabling command execution will change the state of resources in your AWS account, including but not limited to stopping and restarting the service.\n Do you wish to continue?
+ecs.execute_command_disable_warning=<html>Disabling command execution will change the state of resources in your AWS account, including but not limited to stopping and restarting the service.<br> Do you wish to continue?</html>
```

Comment:
i thought a `<br>` would have been sufficient, without the surrounding html

Reply:
No, an html tag is required. It doesnt recognize the <br> tag otherwise